docs/Tutorial_VALMOD.ipynb

```diff
@@ -0,0 +1,1344 @@
+{
```

Comment:
<p>What does the subscript <code>c</code> mean here??  Ohh, "critical point"??</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/TDAmeritrade/stumpy/pull/586/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='58'/>

Reply:
Yes! But, actually, I should have removed subscript `c` as you can see I removed them in the rest of equations. I used it first and then I thought it is not necessary as the reader should know that the values `\mu_prime` and `\sigma_prime` are NOT just ANY values anymore, but in fact, they are the calculated values for the critical point.

What do you think?

Reply:
<p>Sure. That is probably fine</p>